NuGet.config

```diff
@@ -15,6 +15,8 @@
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
+    <!-- Temporary feed for Xamarin workload manifest -->
+    <add key="xamarin" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
```

Comment:
@jonathanpeppers can we put the xamarin packages on one of the existing feeds so we don't have to add this?

Reply:
I guess ideally we would be pushing stuff to this one? 

https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json

I need to find the right person to help us do that.

Reply:
I think @mmitche would be the one to talk to.

Reply:
Probably. @pranavkm is pushing the blazor package to that feed, do you know who could help?

Reply:
Continual upload of those packages, or a one off push?

Reply:
@mmitche I'll follow up over email tomorrow.